dd-sdk-android-navigation/src/main/kotlin/com/datadog/android/androidx/navigation/NavigationViewTrackingStrategy.kt

```diff
@@ -16,9 +16,9 @@ import androidx.navigation.NavDestination
 import androidx.navigation.findNavController
 import androidx.navigation.fragment.DialogFragmentNavigator
 import androidx.navigation.fragment.FragmentNavigator
-import com.datadog.android.rum.ActivityLifecycleTrackingStrategy
 import com.datadog.android.rum.GlobalRum
-import com.datadog.android.rum.ViewTrackingStrategy
+import com.datadog.android.rum.tracking.ActivityLifecycleTrackingStrategy
```

Comment:
Shouldn't we move the navigation stuff in the common `androidx` module ?

Reply:
Good idea

Reply:
Actually ....I don't think so as this can be used without any AndroidX extension. We can discuss about this eventually and move it later in another PR but for now I would keep it like this.